src/pages/generate/pages/result/components/DetectionHotspots.tsx

```
@@ -0,0 +1,325 @@
+    // 2차: 레이블 텍스트 기반 매칭 — finalLabel 과 서버 카테고리명 비교
+    const nameToAllowedId = new Map<string, number>();
+    (allowedCategories ?? []).forEach((c) => {
+      const n = (c.categoryName ?? '').toString().trim();
+      if (!n) return;
+      nameToAllowedId.set(n.toUpperCase(), c.id);
+      nameToAllowedId.set(n.replaceAll(' ', '_').toUpperCase(), c.id);
+    });
+    const fallbackLabels = [hotspot.finalLabel ?? '', hotspot.className ?? '']
+      .flatMap((label) =>
+        label
+          .split('/')
```

ESLint

[error] Replace `'/'` with `"/"`

(prettier/prettier)